eos-tech-support/eos-download-image

```diff
@@ -267,6 +288,10 @@ class EosDownloadImage(object):
                        help='Image version (default: newest)')
         g.add_argument('-I', '--iso', action='store_true',
                        help='Fetch an ISO image, not a raw disk image')
+        g.add_argument('-u', '--url',
+                       help='Fetch compressed image by URL (use with --size)')
+        g.add_argument('-s', '--size',
```

Comment:
Add either `type=int`, or define this higher up the file and add `type=positive_int`

```python
def positive_int(s):
    value = int(s)
    if value <= 0:
        raise argparse.ArgumentTypeError(s + ' is not positive')
    return value
```